pkg/cli/login/helpers.go

```diff
@@ -74,6 +75,13 @@ func dialToServer(clientConfig restclient.Config) error {
 		return err
 	}
 
+	// This is to guide a user who passes the console URL instead of server URL
+	// See https://bugzilla.redhat.com/show_bug.cgi?id=1704827
+	contentType := res.Header.Get("Content-Type")
+	if strings.Contains(contentType, "text/html") {
+		return fmt.Errorf("Response Content-type: found text/html")
```

Comment:
Errors always lower case, don't forget to update the other place too. Also consider using an error constant for this.

Reply:
whoops.  fixed, and added `errInvalidServerURL` type